### PR TITLE
#1533 create new team with 12 members

### LINKED ISSUE #1533: [Seed Data] - Create a New Team with at Least 12 Members
Epic: #976 

Quite frankly, the seed data for the local version of Finishline sucks right now. It has quite a small data size when compared to the production application, so we need to add a lot more data to it for it to be useful. The hope is that with these tickets, you can learn more about how prisma and our backend is setup, and we can get a more robust set of data to test the app locally with.

This ticket is to add at a team with at least 12 members. The team name/ members should be cool and related. For example we have a justice league team rn and all the members are in/related to the justice league.

To do this, you'll need to create users first. Create the user data in users.seed.ts for each user, export them like the other users are, and then create the user in the database in the seed.ts file. After you've created the users, you should create the team. Create the team data in teams.seed.ts. Then in seed.ts create the team and set one of the users as the head under the `  /** Creating Teams */` section. After that you can add the members to the specified team under the `  /** Setting Team Members */` section in the seed.ts file

### PR DESCRIPTION
**Pull request by AR619**
## Changes
-Created a team in teams.ts in seed data 
-Created 12 users in users.ts in seed data
-Added all the users into the team in seed.ts

Made a team and added 12 users as a seed.

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #1533

### CODE REVIEW

**Review by ninest — DISMISSED** [review]

File: src/backend/src/prisma/seed-data/users.seed.ts [hunk 1]
```diff
@@ -308,6 +308,102 @@ const reidChandler: Prisma.UserCreateInput = {
   role: Role.HEAD
 };
 
+const cristianoRonaldo: Prisma.UserCreateInput = {
+  firstName: 'Cristiano',
+  lastName: 'Ronaldo',
+  email: 'cronaldo@united.com',
+  googleAuthId: 'Winger',
+  role: Role.MEMBER
```
Comment (RChandler234) [inline src/backend/src/prisma/seed-data/users.seed.ts:316]: anyone being set as the head of a team should be at least Role.HEAD

**Review by RChandler234 — CHANGES_REQUESTED** [review]
Love it!

**Review by RChandler234 — CHANGES_REQUESTED** [review]
you shouldn't have the .env file committed in this PR

**Review by ninest — APPROVED** [review]

**Review by RChandler234 — APPROVED** [review]
LGTM 🔥

**Pull request merged**